addon/src/modifiers/sortable-item.js

```diff
@@ -833,9 +909,16 @@ export default class SortableItemModifier extends Modifier {
   get height() {
     let el = this.element;
     let height = el.offsetHeight;
+    let elStyles = getComputedStyle(el);
+
+    // This is needed atm only for grid, to fix jumping on drag-start.
+    // In test-app it looks like there is a side-effect when we activate also for direction vertical.
+    // If any user will anytime report a jumping in vertical direction, we should activate for every direction and fix our test-app
+    if (this.direction === 'grid') {
+      height += parseFloat(elStyles.marginTop);
```

Comment:
I have looked shortly in issues.. i think generally adding this could be fix this issue #520, but this should be done in next PR

Reply:
fractional pixels can be a doozy!